### ISSUE TITLE
How to add column break?

### ISSUE THREAD

**Issue by phuwin** [PHPOffice/PHPWord#1053]
How do I add a column break?

I think there is no such function in section class. Would like to add but I cannot find where the section class with addTextBreak or addPageBreak functions are.

**Comment by troosan** [comment]
so far the column break is not supported.
Ideally the PageBreak should be refactored to be just a Break (which can be of different types: page, column, textWrapping)
see http://www.datypic.com/sc/ooxml/e-w_br-1.html

**Comment by tobikummer** [comment]
How is the plan to solve that issue? We are also very interested in this feature!

**Comment by markmorris** [comment]
Yeah i need this feature. Does anyone have any idea of how this might work? I maybe able to implement it.
